### PR TITLE
Remove realm argument of "perform a security check"

### PR DESCRIPTION
**Pull request by annevk**
Fixes https://www.w3.org/Bugs/Public/show_bug.cgi?id=29450.

***
[Preview](https://rawgit.com/heycam/webidl/annevk/security-check/index.html) | [Diff](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fheycam.github.io%2Fwebidl%2F&doc2=https%3A%2F%2Frawgit.com%2Fheycam%2Fwebidl%2Fannevk%2Fsecurity-check%2Findex.html)

### CODE REVIEW

**Comment by annevk** [comment]
Note that we should try landing this together with a change to the HTML Standard that I've yet to prepare.

**Comment by bzbarsky** [comment]
Hmm.  That preview doesn't seem to reflect the changes.  :( @tobie

File: index.bs [hunk 1]
```diff
@@ -9939,19 +9939,14 @@ allowed.  The security check takes the following four inputs:
```
Comment (bzbarsky) [inline index.bs]: Github sucks and won't let me comment on the line I want to comment on, but the security checks now takes _three_ inputs, not four.

**Comment by tobie** [comment]
> Hmm. That preview doesn't seem to reflect the changes. :( @tobie

Wow. No idea what's going on here. That's weird.

**Comment by tobie** [comment]
> Hmm. That preview doesn't seem to reflect the changes. :( @tobie

OK, filed a support ticket with GH. It's an issue on their side due to an update in their markdown engine. Should be fixed soon.

**Comment by tobie** [comment]
Oh, but there are multiple issues, here.

The other issue, of course, is that @annevk didn't add the rendered Bikeshed content to his PR.

**Comment by annevk** [comment]
I did not because updated bikeshed emits a bunch of errors. Happy to update once those are sorted.

**Comment by tobie** [comment]
> I did not because updated bikeshed emits a bunch of errors. Happy to update once those are sorted.

@annevk: is that a BS issue, or something to do with the WebIDL src file I should be fixing?

**Comment by annevk** [comment]
The latter for the most part I think. It got stricter about some things.

**Comment by tobie** [comment]
Fixed the bikeshed issues (thanks to Tab). Rebased and rendered your PR. Preview should be OK, now.

**Comment by annevk** [comment]
I wanted to bump the date before landing, but I don't think the bs issues have been resolved. They almost certainly require changes to `index.bs`.

**Comment by domenic** [comment]
They do not; it turned out to be only Bikeshed-side issues where Bikeshed was seeing collisions between Web IDL and Web IDL level 1.

**Comment by annevk** [comment]
With latest bikeshed (git pull, bikeshed update) I get:
```
LINK ERROR: Ambiguous for-less link, please see <https://tabatkins.github.io/bikeshed/#ambi-for> for instructions:
<a data-link-type="dfn" data-lt="flattened member types">flattened member types</a>
LINK ERROR: Ambiguous for-less link, please see <https://tabatkins.github.io/bikeshed/#ambi-for> for instructions:
<a data-link-type="dfn" data-lt="takes an argument list">take an argument list</a>
LINK ERROR: Ambiguous for-less link, please see <https://tabatkins.github.io/bikeshed/#ambi-for> for instructions:
<a data-link-type="dfn" data-lt="member types">member types</a>
LINK ERROR: Ambiguous for-less link, please see <https://tabatkins.github.io/bikeshed/#ambi-for> for instructions:
<a data-link-type="dfn" data-lt="takes a named argument list">named argument list</a>
LINK ERROR: Ambiguous for-less link, please see <https://tabatkins.github.io/bikeshed/#ambi-for> for instructions:
<a data-link-type="dfn" data-lt="takes a named argument list">named argument lists’</a>
LINK ERROR: Ambiguous for-less link, please see <https://tabatkins.github.io/bikeshed/#ambi-for> for instructions:
<a data-link-type="dfn" data-lt="member type">member type</a>
LINK ERROR: Ambiguous for-less link, please see <https://tabatkins.github.io/bikeshed/#ambi-for> for instructions:
<a data-link-type="dfn" data-lt="created">create</a>
LINK ERROR: Ambiguous for-less link, please see <https://tabatkins.github.io/bikeshed/#ambi-for> for instructions:
<a data-link-type="dfn" data-lt="Throw">Throw</a>
LINK ERROR: Ambiguous for-less link, please see <https://tabatkins.github.io/bikeshed/#ambi-for> for instructions:
<a data-link-type="dfn" data-lt="created">created</a>
LINK ERROR: Ambiguous for-less link, please see <https://tabatkins.github.io/bikeshed/#ambi-for> for instructions:
<a data-link-type="dfn" data-lt="member types">union member type</a>
LINK ERROR: Ambiguous for-less link, please see <https://tabatkins.github.io/bikeshed/#ambi-for> for instructions:
<a data-link-type="dfn" data-lt="detach">detached</a>
LINK ERROR: Ambiguous for-less link, please see <https://tabatkins.github.io/bikeshed/#ambi-for> for instructions:
<a data-link-type="dfn" data-lt="takes no arguments">take no arguments</a>
LINK ERROR: Ambiguous for-less link, please see <https://tabatkins.github.io/bikeshed/#ambi-for> for instructions:
<a data-link-type="dfn" data-lt="takes an identifier">take an identifier</a>
LINK ERROR: Ambiguous for-less link, please see <https://tabatkins.github.io/bikeshed/#ambi-for> for instructions:
<a data-link-type="dfn" data-lt="takes a named argument list">take a named argument list</a>
LINK ERROR: Ambiguous for-less link, please see <https://tabatkins.github.io/bikeshed/#ambi-for> for instructions:
<a data-link-type="dfn" data-lt="takes an identifier">take the identifier</a>
LINK ERROR: Ambiguous for-less link, please see <https://tabatkins.github.io/bikeshed/#ambi-for> for instructions:
<a data-link-type="dfn" data-lt="flattened member types">flattened members</a>
LINK ERROR: Ambiguous for-less link, please see <https://tabatkins.github.io/bikeshed/#ambi-for> for instructions:
<a data-link-type="dfn" data-lt="thrown">thrown</a>
LINK ERROR: Ambiguous for-less link, please see <https://tabatkins.github.io/bikeshed/#ambi-for> for instructions:
<a data-link-type="dfn" data-lt="created">creating</a>
WARNING: This specification has neither a 'Security Considerations' nor a 'Privacy Considerations' section. Please consider adding both, see https://w3ctag.github.io/security-questionnaire/.
 ✔  Successfully generated, with 18 linking errors
```

**Comment by tobie** [comment]
@annevk: did you run `$ bikeshed update`?

**Comment by tobie** [comment]
Oh. Never mind. You said you did.

**Pull request merged**